browser/src/control/Control.AutoFillOptions.ts

```diff
@@ -113,7 +113,10 @@ class AutoFillOptions extends L.Control.AutoCompletePopup {
 			}
 
 		} else if (eventType === 'keydown') {
-			// TODO: handle arrowDown key
+			if (object.key !== 'Tab' && object.key !== 'Shift') {
+				this.map.focus();
+				return true;
+			}
```

Comment:
it shouldn't be much different that in the base class - can we call the base class method?

Reply:
I moved `if (object.key !== 'Tab' && object.key !== 'Shift')` to the base class.